SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +23,31 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+BuildRequires:  kernel-devel
+BuildRequires:  kmod
 Requires:       python3
+Requires:       kernel
+
+# set package name
+%{!?_name: %global _name mstflint}
+
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global install_mod_dir extra/%{_name}
 
 %description
 This package contains firmware update tool, vpd dump and register dump tools
 for network adapters based on Mellanox Technologies chips.
 
+%package kernel
+Summary:        mstflint Kernel Modules
+Group:          System Environment/Kernel
+Requires:       kernel-headers >= 5.15.0
```

Comment:
What is the reason for restricting the kernel version?
As far as I know, mstflint supports kernel versions 4.x as well. Is it something related to Microsoft?

Reply:
I think I agree with you. The reason for that limit is that Mariner 2.0 uses kernel 5.15.x and we wanted to limit the support scope of this OOT module.